content/en-us/reference/engine/classes/SoundService.yaml

```
@@ -86,8 +86,29 @@ properties:
     capabilities: []
     writeCapabilities: []
   - name: SoundService.DefaultListenerLocation
-    summary: ''
-    description: ''
+    summary: |
+      Determines where (if anywhere) to place an `Class.AudioListener` by default.
+    description: |
+      Determines where to place an `Class.AudioListener` by default.
+      If `Class.SoundService.DefaultListenerLocation` is `None`, then no `Class.AudioListener` will be created by default, but `Class.AudioListeners` can be created separately by scripts.
+      If `Class.SoundService.DefaultListenerLocation` is `Camera`, then
+      - an `Class.AudioListener` will be created and parented-to `Class.Workspace.CurrentCamera`
+      - an `Class.AudioDeviceOutput` will be created and parented-to `Class.SoundService`
+      - a `Class.Wire` will be created and parented-to the previously-created `Class.AudioListener`
+      - `Class.Wire.SourceInstance` will be set to the previously-created `Class.AudioListener`, and `Class.Wire.TargetInstance` will be set to the previously-created `Class.AudioDeviceOutput`
+      So, the world will be heard from the perspective (postition and orientation) of your camera.
```

The "so," seems overly casual to me.
I might say something like "The result is".

---

Sure thing; does this work?
```suggestion
      As a result, the world will be heard from the perspective (postition and orientation) of your camera.
```